okio-fakefilesystem/src/commonMain/kotlin/okio/fakefilesystem/FakeFileSystem.kt

```diff
@@ -71,6 +71,9 @@ class FakeFileSystem(
   val clock: Clock = Clock.System,
 ) : FileSystem() {
 
+  // Avoid propagating 'ExperimentalTime' if 'clock' is not specified.
+  constructor() : this(Clock.System)
```

Comment:
Not sure if it's worth maintaining this secondary constructor forever to avoid propagating `@ExperimentalTime` to `FakeFileSystem()`. Let me know what you think